app/api/entities.rb

```diff
@@ -25,12 +25,35 @@ class Office < Grape::Entity
     expose :address
   end
 
+  class DeputyLegislature < Grape::Entity
+    expose :legislative_initiatives_count, as: :legislative_initiatives
+    expose :signed_motions_count, as: :signed_motions
+    expose :speeches_count, as: :speeches
+    expose :draft_decisions_count, as: :draft_decisions
+    expose :questions_count, as: :questions
+    expose :legislature, using: Legislature
+    expose :electoral_circumscription, using: ElectoralCircumscription
+  end
+
   class Deputy < Grape::Entity
     expose :id, documentation: { type: Integer }
     expose :image_link
     expose :email
     expose :name
+    expose :date_of_birth
+    expose :room
+  end
+
+  class DeputyParty < Grape::Entity
+    expose :start_date
+    expose :end_date
+    expose :party, using: Party
+  end
+
+  class FullDeputy < Deputy
     expose :offices, using: Office
+    expose :deputy_parties, using: DeputyParty, as: :parties, documentation: { is_array: true }
```

Comment:
Still thinking about what @petrovicim needs on his side and if we move this under the `Deputy` entity or leave it here. For now, maybe we can go on with the current implementation and we'll double-check later.